.github/bug-report.yml

```diff
@@ -0,0 +1,76 @@
+# Source: https://github.com/arduino/tooling-project-assets/blob/main/issue-templates/forms/platform-dependent/bug-report.md
```

Comment:
```suggestion
# Source: https://github.com/arduino/tooling-project-assets/blob/main/issue-templates/forms/platform-dependent/bug-report.yml
```

This is an error I made in the upstream copy. I think it is easier to fix it here from the start rather than via a separate sync PR later.

I have it on my "to do" list to submit a fix upstream for this and several other problems in the documentation comments of the issue templates but didn't find the time to finish it yet